src/containers/MoveListView.js

```diff
@@ -138,19 +148,29 @@ class MoveListView extends React.Component {
 
 	}
 
+
+	tabsChange = (key) => {
+		this.setState({ 
+			selected_move: null,
+			selected_move_idx: -1,
+			current_tab: key,
+		})
+	}
+
 	render() {
 		return (
-			<Tabs defaultActiveKey="1" onChange={console.log(this.key)}>
+			<Tabs defaultActiveKey="1" onChange={(key) => this.tabsChange(key)}>
```

Comment:
Can you change it to the thing we talked about. Like define the tab names in the const array above and then use the elements of the array as the keys rather than "1".